ironic-deployment/ironic/ironic.yaml

```diff
@@ -42,20 +42,6 @@ spec:
                 secretKeyRef:
                   name: mariadb-password
                   key: password
-        - name: ironic-httpd
-          image: quay.io/metal3-io/ironic
-          imagePullPolicy: Always
-          securityContext:
-             capabilities:
-               add: ["NET_ADMIN"]
-          command:
-            - /bin/runhttpd
-          volumeMounts:
-            - mountPath: /shared
-              name: ironic-data-volume
-          envFrom:
-            - configMapRef:
-                name: ironic-bmo-configmap
```

Comment:
you should not remove this. This is the webserver for IPA image and other PXE related files

Reply:
@maelk I remove it because the httpd server which works as a WSGI server inside the ironic-api pod will also serve the IPA image.